spark-indexer/src/test/resources/2020_02/uniprotkb/idtracker4uuw.txt

```diff
@@ -0,0 +1,11 @@
+Q9EPI6	NSMFOLD1_RAT
+Q9EPI6	NSMFOLD2_RAT
+Q9EPI6	NSMFOLD3_RAT
+A0A009IHW8	ABTIR_ACIBA
+A0A011QK89	L2HDH_ACCPH
+A0A011QK89	L2HDH_ACCSB
+A0A023FFB5	E1181_AMBCJ
+A0A024R780	MRI_HUMAN
+A0A059WLZ7	CNDA_SPHSD
+A0A075B6S8	KV110_HUMAN
+A0A077YBL0	PDEG_PLAYO
```

Comment:
Please add one example of unreviewed entry where this condition is satisfied
// don't add first component for trembl entries, since this is the accession, and
                // we do not want false boosting for default searches that match a substring of
                // the accession